src/nginxconfig/templates/domain_sections/php.vue

```diff
@@ -219,6 +314,69 @@ THE SOFTWARE.
                 },
                 deep: true,
             },
+            // Check server selection is valid
+            '$props.data.phpServer': {
+                handler(data) {
+                    if (data.enabled) {
+                        // This might cause recursion, but seems not to
+                        if (!Object.keys(data.options).includes(data.computed))
+                            data.computed = data.default;
+
+                        // Show the custom box
+                        this.$props.data.phpServerCustom.enabled = data.computed === 'custom';
+                        return;
+                    }
+
+                    // Hide custom if disabled
+                    this.$props.data.phpServerCustom.enabled = false;
+                },
+                deep: true,
+            },
+            // Check backup server selection is valid
+            '$props.data.phpBackupServer': {
+                handler(data) {
+                    if (data.enabled) {
+                        // This might cause recursion, but seems not to
+                        if (!Object.keys(data.options).includes(data.computed))
+                            data.computed = data.default;
+
+                        // Show the custom box
+                        this.$props.data.phpBackupServerCustom.enabled = data.computed === 'custom';
+                        return;
+                    }
+
+                    // Hide custom if disabled
+                    this.$props.data.phpBackupServerCustom.enabled = false;
+                },
+                deep: true,
+            },
+            // Enable PHP server settings if any site uses PHP
+            '$parent.$parent.$data.domains': {
+                handler(data) {
+                    for (const domain of data) {
+                        if (domain && domain.php && domain.php.php && domain.php.php.computed) {
+                            this.$props.data.phpServer.enabled = true;
+                            this.$props.data.phpServer.computed = this.$props.data.phpServer.value;
+                            this.$props.data.phpBackupServer.enabled = true;
+                            this.$props.data.phpBackupServer.computed = this.$props.data.phpBackupServer.value;
+                            return;
+                        }
+                    }
+                    this.$props.data.phpServer.enabled = false;
+                    this.$props.data.phpServer.computed = '';
+                    this.$props.data.phpBackupServer.enabled = false;
+                    this.$props.data.phpBackupServer.computed = '';
+                },
+                deep: true,
+            },
```

Comment:
```suggestion
```
This watcher is not needed, this is now site-specific, we don't need to be checking other domains.

However, you do need to update the `// Disable everything if PHP is disabled` watcher above to also disable `phpServer` and `phpBackupServer` as well as all the rules options it currently disables.